packages/skeleton-react/src/lib/components/Avatar/schema.json

```diff
@@ -1,164 +1,189 @@
-					"type": "string"
-				},
-				"imageClasses": {
-					"description": "Provide avatar image arbitrary CSS classes.",
-					"type": "string"
-				},
-				"rounded": {
-					"description": "Set border radius styles.",
-					"type": "string"
-				},
-				"shadow": {
-					"description": "Set shadow styles.",
-					"type": "string"
-				},
-				"size": {
-					"description": "Set size styles.",
-					"type": "string"
-				},
-				"src": {
-					"description": "Set avatar image source URL.",
-					"type": "string"
-				}
-			},
-			"propertyOrder": [
-				"src",
-				"alt",
-				"filter",
-				"base",
-				"background",
-				"size",
-				"font",
-				"border",
-				"rounded",
-				"shadow",
-				"classes",
-				"imageBase",
-				"imageClasses",
-				"children"
-			],
-			"type": "object"
-		},
-		"Iterable<React.ReactNode>": {
-			"propertyOrder": ["__@iterator@83"],
-			"type": "object"
-		},
-		"React.ReactElement<any,string|React.JSXElementConstructor<any>>": {
-			"description": "Represents a JSX element.\n\nWhere {@link ReactNode} represents everything that can be rendered, `ReactElement`\nonly represents JSX.",
-			"properties": {
-				"key": {
-					"type": "string"
-				},
-				"props": {
-					"description": "The type of the props object"
-				},
-				"type": {
-					"anyOf": [
-						{
-							"propertyOrder": [],
-							"type": "object"
-						},
-						{
-							"propertyOrder": [],
-							"type": "object"
-						},
-						{
-							"type": "string"
-						}
-					],
-					"description": "The type of the component or tag"
-				}
-			},
-			"propertyOrder": ["type", "props", "key"],
-			"required": ["key", "props", "type"],
-			"type": "object"
-		},
-		"React.ReactNode": {
-			"anyOf": [
-				{
-					"$ref": "#/definitions/React.ReactElement<any,string|React.JSXElementConstructor<any>>"
-				},
-				{
-					"$ref": "#/definitions/Iterable<React.ReactNode>"
-				},
-				{
-					"$ref": "#/definitions/React.ReactPortal"
-				},
-				{
-					"type": ["string", "number", "boolean"]
-				}
-			],
-			"description": "Represents all of the things React can render.\n\nWhere {@link ReactElement} only represents JSX, `ReactNode` represents everything that can be rendered."
-		},
-		"React.ReactPortal": {
-			"properties": {
-				"children": {
-					"$ref": "#/definitions/React.ReactNode"
-				},
-				"key": {
-					"type": "string"
-				},
-				"props": {
-					"description": "The type of the props object"
-				},
-				"type": {
-					"anyOf": [
-						{
-							"propertyOrder": [],
-							"type": "object"
-						},
-						{
-							"propertyOrder": [],
-							"type": "object"
-						},
-						{
-							"type": "string"
-						}
-					],
-					"description": "The type of the component or tag"
-				}
-			},
-			"propertyOrder": ["children", "type", "props", "key"],
-			"required": ["children", "key", "props", "type"],
-			"type": "object"
-		}
-	}
+    "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
Not sure I follow. These are the source of the data shown on the doc pages. Without the schema there's no data and nothing to display. You may be thinking the tables are cloned from this data, but that's not currently the case.

Either way this would be out of scope of this PR.

Reply:
Yeah out of scope, but build artifacts shouldn't be pushed to a repo. These are only neccesary when building the website.  So they shouldn't be pushed and instead be gitignored. Just like how you gitignore `/build`.